include/OpenSpaceToolkit/Astrodynamics/Access.hpp

```diff
@@ -27,83 +27,78 @@ namespace astro
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-using ostk::core::types::String ;
-using ostk::core::ctnr::Array ;
+using ostk::core::ctnr::Array;
+using ostk::core::types::String;
 
-using ostk::physics::time::Instant ;
-using ostk::physics::time::Duration ;
-using ostk::physics::time::Interval ;
+using ostk::physics::time::Duration;
+using ostk::physics::time::Instant;
+using ostk::physics::time::Interval;
 
-using ostk::physics::units::Angle ;
+using ostk::physics::units::Angle;
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 /// @brief                      Object-to-object visibility
 
 class Access
 {
+   public:
+    enum class Type
+    {
 
-    public:
-
-        enum class Type
-        {
-
-            Undefined,
-            Complete,
-            Partial
-
-        } ;
-
-                                Access                                      (   const   Access::Type&               aType,
-                                                                                const   Instant&                    anAcquisitionOfSignal,
-                                                                                const   Instant&                    aTimeOfClosestApproach,
-                                                                                const   Instant&                    aLossOfSignal,
-                                                                                const   Angle&                      aMaxElevation                               ) ;
+        Undefined,
+        Complete,
+        Partial
 
-        bool                    operator ==                                 (   const   Access&                     anAccess                                    ) const ;
+    };
 
-        bool                    operator !=                                 (   const   Access&                     anAccess                                    ) const ;
+    Access(const Access::Type& aType,
+           const Instant& anAcquisitionOfSignal,
+           const Instant& aTimeOfClosestApproach,
+           const Instant& aLossOfSignal,
+           const Angle& aMaxElevation);
```

Comment:
Did we consider a style like this? (and is this even configurable via the formatter?)

```suggestion
    Access(
           const Access::Type& aType,
           const Instant& anAcquisitionOfSignal,
           const Instant& aTimeOfClosestApproach,
           const Instant& aLossOfSignal,
           const Angle& aMaxElevation
    );
```

Reply:
I don't think so but let me confirm. The parameters for that should be AlignAfterOpenBracket and BinPackArguments
BinPackParameters

Reply:
So this was configurable by setting the style to Allman